.github/dependabot.yaml

```diff
@@ -5,3 +5,14 @@ updates:
     schedule:
       interval: "daily"
       time: "12:00"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Should we change this to monthly? I know, it's a trade-off with getting security fixes quickly, but we don't plan to make any new changes in this repository or releases, and clicking through renovate PRs also takes some time (every week).